services/cptec/cities.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { transform } from 'camaro';
+import { CPTEC_URL, CITY_TEMPLATE } from './constants';
+
+/**
+ * Get all Brazilian cities on CPTEC database
+ * @returns {Array}
+ */
+export const getAllCitiesData = async () => {
+  const citiesData = await axios.get(`${CPTEC_URL}listaCidades`, {
```

Comment:
Aqui talvez seja mais uma questão de gosto, mas geralmente utilizo como no exemplo abaixo para ficar mais claro
```suggestion
  const citiesData = await axios.get(`${CPTEC_URL}/listaCidades`, {
```

Reply:
Tá ai, também gosto mais... rs
Feito.